server/handlers/likeHandler.ts

```diff
@@ -0,0 +1,44 @@
+import { CreateLikeRequest, CreateLikeResponse, GetLikesRequest, GetLikesResponse,  } from '../api';
+import { db } from '../datastore';
+import { getUserId } from '../localStorage';
+import { ExpressHandler, Like } from '../types';
+
+export const createLikeHandler: ExpressHandler<CreateLikeRequest, CreateLikeResponse> = async(
+  req,
+  res
+) => {
+  //TODO:Get Userid from Session
+  if (!req.body.postId) {
+    return res.status(400).send({error:'No Post Id'});
+  }
+
+  if (!getUserId()) {
```

Comment:
similar to above, let's get it from the request body for now until I add extracting it from the jwt token.